### PR TITLE
sc-im: new package at 0.7.0

### PR DESCRIPTION
**Pull request by eudaldgr**
## Description of package

[SC-IM - Spreadsheet Calculator Improvised](https://github.com/andmarti1424/sc-im)

## Installed manifest of package

```
/var/db/kiss/installed/sc-im/version
/var/db/kiss/installed/sc-im/sources
/var/db/kiss/installed/sc-im/manifest
/var/db/kiss/installed/sc-im/depends
/var/db/kiss/installed/sc-im/checksums
/var/db/kiss/installed/sc-im/build
/var/db/kiss/installed/sc-im/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/scim/scim_help
/usr/share/scim/plot_scatter
/usr/share/scim/plot_pie
/usr/share/scim/plot_line
/usr/share/scim/plot_bar
/usr/share/scim/
/usr/share/man/man1/scim.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/scim
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by dylanaraps** [comment]
Why does it create `/usr/local/`?

**Comment by eudaldgr** [comment]
> Why does it create `/usr/local/`?

This was created before
```
mv -f "$1/usr/local/bin" "$1/usr/bin"
mv -f "$1/usr/local/share" "$1/usr/share"
```

**Comment by dylanaraps** [comment]
Wait. Aren't these the same thing? (This and `sc`)? Is this not a continuation of `sc`? If so, why include both?

File: community/sc-im/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+cd src
+
+make
+make DESTDIR="$1" PREFIX=/usr install
```
Comment (dylanaraps) [inline community/sc-im/build]: Remove the `cd` and use `-C src` for both `make` commands.

**Comment by eudaldgr** [comment]
> Wait. Aren't these the same thing? (This and `sc`)? Is this not a continuation of `sc`? If so, why include both?

This is like vi, vim and neovim.
`scim` it's `sc improved`

**Comment by dylanaraps** [comment]
Ah.

**Comment by eudaldgr** [comment]
I use `scim`, but maybe some people wants the classic one `sc`.

File: community/sc-im/depends [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bison
```
Comment (dylanaraps) [inline community/sc-im/depends]: `bison make`? Is bison needed at runtime or only build time?

Comment (eudaldgr) [reply community/sc-im/depends]: `bison make` it's the answer.

File: community/sc-im/build (hunk 1 above)
Comment (eudaldgr) [reply community/sc-im/build]: Done.

**Pull request merged**